node/src/accountant/mod.rs

```diff
@@ -1516,21 +1550,23 @@ mod tests {
         assert_eq!(system.run(), 0);
         let after = SystemTime::now();
         let mut adjust_payments_params = adjust_payments_params_arc.lock().unwrap();
-        let (cwbqp_msg, captured_now, logger_clone) = adjust_payments_params.remove(0);
+        let (actual_awaited_adjustment, captured_now, logger_clone) =
+            adjust_payments_params.remove(0);
+        assert!(adjust_payments_params.is_empty());
```

Comment:
Moving this back down to the end of this batch of assertions will give better failure messages.